RecoLocalMuon/Configuration/python/RecoLocalMuon_EventContent_cff.py

```diff
@@ -28,13 +28,11 @@
         'keep *_cscSegments_*_*', 
         'keep *_rpcRecHits_*_*')
 )
-
-def _modifyRecoLocalMuonEventContentForPhase2( object ):
-    object.outputCommands.append('keep *_gemRecHits_*_*')
-    object.outputCommands.append('keep *_me0RecHits_*_*')
-    object.outputCommands.append('keep *_me0Segments_*_*')
-
 from Configuration.StandardSequences.Eras import eras
-eras.phase2_muon.toModify( RecoLocalMuonFEVT, func=_modifyRecoLocalMuonEventContentForPhase2 )
-eras.phase2_muon.toModify( RecoLocalMuonRECO, func=_modifyRecoLocalMuonEventContentForPhase2 )
-eras.phase2_muon.toModify( RecoLocalMuonAOD,  func=_modifyRecoLocalMuonEventContentForPhase2 )
+def _updateOutput( era, outputPSets, commands):
```

Comment:
its in fact this same one.. RecoLocalMuon/Configuration/python/RecoLocalMuon_EventContent_cff.py - we hopefully can avoid era statements in the event content


> On Apr 29, 2016, at 4:19 PM, Sven Dildick <notifications@github.com> wrote:
> 
> In RecoLocalMuon/Configuration/python/RecoLocalMuon_EventContent_cff.py:
> 
> >  from Configuration.StandardSequences.Eras import eras
> > -eras.phase2_muon.toModify( RecoLocalMuonFEVT, func=_modifyRecoLocalMuonEventContentForPhase2 )
> > -eras.phase2_muon.toModify( RecoLocalMuonRECO, func=_modifyRecoLocalMuonEventContentForPhase2 )
> > -eras.phase2_muon.toModify( RecoLocalMuonAOD,  func=_modifyRecoLocalMuonEventContentForPhase2 )
> > +def _updateOutput( era, outputPSets, commands):
> 
> @davidlange6 Which file is that?
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
>